app/controllers/sdg_management/relations_controller.rb

```diff
@@ -19,7 +20,11 @@ def edit
   def update
     @record.sdg_target_list = params[@record.class.table_name.singularize][:sdg_target_list]
 
-    redirect_to action: :index
+    if @record.sdg_revision.blank? && @record.create_sdg_revision
+      flash[:notice] = t("sdg_management.relations.update.notice", relatable: relatable_class.model_name.human)
+    end
```

Comment:
I actually forgot to add the `flash[:notice]` in this action :sweat_smile:. So how about:

```ruby
@record.create_sdg_revision! unless @record.sdg_revision
flash[:notice] = t("sdg_management.relations.update.notice", relatable: relatable_class.model_name.human)
```

And I guess we should update the flash message to be more generic :thinking:.

Reply:
As I am redirecting the user always to the "Pending" tab after the update action I intended to show a more precise notice when the SDG manager is marking the relatable as revised (this happens when the user updates for the first time the relatable from SDG management section) so the user knows that the relatable was marked as revised and moved into "Marked as revised" tab. What do you think about having two notices?

1º Notice for the first relatable update: Proposal was marked as revised and its SDG relations were updated successfully!
2º Notice for subsequent updates: SDGrelations were updated successfully!

The action code could look like:
```
  def update
    @record.sdg_target_list = params[@record.class.table_name.singularize][:sdg_target_list]

    if @record.sdg_revision.blank?
      @record.create_sdg_revision!
      flash[:notice] = t("sdg_management.relations.update_and_revise.notice",
        relatable: relatable_class.model_name.human)
    else
      flash[:notice] = t("sdg_management.relations.update.notice",
        relatable: relatable_class.model_name.human)
    end
    redirect_to action: :index
  end
```

Reply:
Ping @javierm, just in case you missed it.